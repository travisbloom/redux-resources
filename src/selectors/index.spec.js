import {
    INITIAL_STATE_WITH_CACHED_RESOURCE_AND_CACHED_RESOURCE_LIST,
    QUERY_STRING,
    RESOURCE_REDUCER,
} from '../mocks'

import { selectAndDenormalizeResourceList } from './'

it('will update the state after the success action', async () => {
    const selector = selectAndDenormalizeResourceList(RESOURCE_REDUCER)
    expect(selector(
        INITIAL_STATE_WITH_CACHED_RESOURCE_AND_CACHED_RESOURCE_LIST,
        QUERY_STRING
    )).toMatchSnapshot()
})
