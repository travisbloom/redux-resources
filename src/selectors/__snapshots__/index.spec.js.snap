exports[`test will update the state after the success action 1`] = `
Array [
  Object {
    "id": "resourceId",
    "prop1": "initial prop1",
    "prop2": "initial prop2",
    "relatedResource": "relatedResourceId",
  },
]
`;
